docs/package.json

```diff
@@ -14,12 +14,13 @@
     "docs:lint:fix": "eslint --ext .js,.vue,.mjs .vuepress content --fix",
     "docs:scripts:link-assets": "node .vuepress/tools/link-assets.mjs",
     "docs:scripts:generate-legacy-docs-versions": "node .vuepress/tools/generate-legacy-docs-versions.mjs",
-    "docs:scripts:generate-js-examples": "bash .vuepress/tools/generate-js-examples.sh",
+    "docs:code-examples:generate-js": "bash .vuepress/tools/code-examples-generator.sh",
+    "docs:code-examples:generate-all-js": "bash .vuepress/tools/code-examples-generator.sh --generateAll",
+    "docs:code-examples:format-all-ts": "bash .vuepress/tools/code-examples-generator.sh --formatAllTsExamples",
     "docs:start": "npm run docs:scripts:link-assets && cross-env NODE_OPTIONS=--openssl-legacy-provider node ./node_modules/.bin/vuepress dev ./",
     "docs:start:no-cache": "npm run docs:scripts:link-assets && cross-env NODE_OPTIONS=--openssl-legacy-provider node ./node_modules/.bin/vuepress dev --no-cache ./",
     "docs:serve": "http-server -c-1 .vuepress/dist/",
-    "docs:build": "npm run docs:scripts:generate-js-examples && npm run docs:build:no-examples",
-    "docs:build:no-examples": "cross-env NODE_OPTIONS=--openssl-legacy-provider node .vuepress/tools/build.mjs -- --no-cache",
```

Comment:
Are we sure we want to remove this one?

Reply:
We don't need it anymore since I remove the build examples step from the normal build script